frontend/src/css/main.less

```diff
@@ -6,25 +6,28 @@ header {
 }
 
 main {
-  margin: 2em;
-  padding: 1em;
-  border: 1px solid @nice-blue;
   display: inline-block;
-}
 
-.show-selector {
-  display: inline-block;
-  vertical-align: top;
-  width: 10em;
+  .shopping-stage {
+    border: 1px solid @nice-blue;
+    float: left;
+    display: inline-block;
+    vertical-align: top;
+    min-width: 10em;
+    height: 200px;
```

Comment:
The height setting doesn't work well with the longer list of shows in e.g. master branch. Maybe min-height?